### PR TITLE
Adjustments for IO refactoring

### PR DESCRIPTION
**Pull request by malmaud**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +1,3 @@
-julia 0.4-
+julia 0.5-
```
Comment (tkelman) [inline REQUIRE]: are we going to maintain separate branches or is this just for testing?

**Comment by malmaud** [comment]
I was thinking separate branches.

**Comment by tkelman** [comment]
That could get messy. If we can manage to get all of this via compat that might be less ongoing maintenance burden.

**Comment by malmaud** [comment]
Sure. I probably won't look into that myself for some time, though.

**Comment by Keno** [comment]
Can we get this merged? Requests seems broken for HTTPS connections without this.

**Comment by malmaud** [comment]
I’ll work on getting the necessary changes to Compat merged today so that we can merge this without needing separate branches.

**Pull request merged**